src/service/data_products/samples.py

```diff
@@ -117,6 +117,7 @@ def _remove_keys(doc):
     doc = remove_collection_keys(remove_arango_keys(doc))
     doc.pop(names.FLD_MATCHES_SELECTIONS, None)
     doc.pop(names.FLD_SAMPLE_GEO, None)
+    doc.pop(names.FLD_KBASE_IDS, None)
```

Comment:
can you remind me why we are removing `FLD_SAMPLE_GEO` and `FLD_KBASE_IDS`?

Reply:
GEO is removed because we already have lat / lon so it's redundant. It's just there so we could do geosearch later if we wanted.

I'm removing IDS because there could be hundreds of IDS in that field and so it gets really big and isn't particularly useful for users. We probably need to have a discussion about if / how we want to display the genome <-> sample relationships to users. If we do decide to keep the list it's a 1 liner to put back in.

Reply:
👍